src/apps/config/leader.lua

```diff
@@ -421,18 +435,19 @@ function Leader:update_configuration (update_fn, verb, path, ...)
    local to_restart =
       self.support.compute_apps_to_restart_after_configuration_update (
          self.schema_name, self.current_configuration, verb, path,
-         self.current_in_place_dependencies, ...)
+         self.in_place_dependencies, ...)
    local new_config = update_fn(self.current_configuration, ...)
-   local new_app_graph = self.setup_fn(new_config)
-   local actions = self.support.compute_config_actions(
-      self.current_app_graph, new_app_graph, to_restart, verb, path, ...)
-   self:enqueue_config_actions(actions)
-   self.current_app_graph = new_app_graph
+   local new_graph = self.setup_fn(new_config, ...)
+   for id, follower in pairs(self.followers) do
```

Comment:
Probably here we should be noticing new followers in the `new_graphs`, starting them, and stopping old followers that are no longer configured.  WDYT?

Reply:
Sure. I was going to be doing that in a different PR but i can do the work and lump it in this one if you'd like.